doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
+#########################
+
+Many packages in Refkit come with a default set of configuration files.
+This document gives guidance for configuring a Refkit image with custom
+package configuration files.
+
+Using .bbappend files to override package configuration
+=======================================================
+
+If you have only a limited set of packages which you need to configure,
+you can create ``.bbappend`` files which append the recipes for each
+package which you want to modify. A usual way to change the
+configuration is to change the install phase to replace or remove the
+default configuration files.
+
+As explained in `yocto-compatible.rst <yocto-compatible.rst>`_, the
```

Comment:
"Yocto Compatible" is a certificate that layers can apply for, but don't have to. It is meant as a mark of quality for layers that are meant to be used by others. It makes no sense for private or product-specific layers.